patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,84 @@
+- Unsure if the processes around the project are conducive to InnerSource style development.
+
+## Sketch (optional)
+
+## Solutions
+
+Based on the two previous works mentioned earlier (see [Rationale](#rationale-optional)), we hypothesize that the _fitness_ of a project can be measured by assessing whether the project has the necessary qualities to attract enough developers to build a community around it, that the tools and processes allow for collaborative development, and that the project maintainers are ready to face the challenges of InnerSource style development. We label this as the  _Project-Process-People model_. We suggest two sub-factors within each which overall provides __six__ dimensions to measure the fitness of candidate projects.
+
+### List of the major factors and the sub-factors
+
+1. __Project Compatibility__
+    - _Seed Product_ -> Do the project characteristics make it suitable for InnerSource style development?
+    - _Multiple Potential Stakeholders_ -> Is the project likely to attract developers from different teams and build a community around the project?
+2. __Process Compatibility__
+    - _Appropriate Tools and Practices_ -> Do project practices facilitate contributions from outsiders and are the tools used in the project too specific and need to learned?
+    - _Ease of contribution_ -> Are the required ancillary resources set up and is the code modular enough to facilitate contribution?
+3. __People Compatibility__
+    - _Team Readiness for InnerSource_ -> Are the team members aware of and ready for the challenges of maintaining an InnerSource project?
+    - _Management Support_ -> Is the management willing to support the developers working on the project and the initiative in general?
+
+As one specific implementation of the pattern, we present a tool for estimating the _fitness_ of a candidate project along the six dimensions, and also present an an aggregate score which is a weighted sum of the scores for the dimensions (weights are determined by how many people voted a characteristic as important in the [2020 InnerSource Survey](https://tapjdey.github.io/InnerSource_Survey_2020/index.html)).
+The tool is written in __R__, and can be run from the [GitHub repository](https://github.com/tapjdey/InnerSource_Project_Fitness) (recommended - instructions in the repository) or from an [online instance](https://tapjdey.shinyapps.io/InnerSource_Fitness_Checker/). The tool is available in Chinese and English (language can be changed on the fly).
+
+## Resulting Context
+
+In general, application of the framework should help assess the InnerSource-readiness of individual projects.
+
+The _fitness score_ of a project can be used as an indication about its suitability for InnerSource style development, which can help in selecting the most appropriate candidate project(s).
```

Comment:
```suggestion
The _fitness score_ of a project can be used as an indication about its suitability for InnerSource style development, which can help in selecting the most appropriate candidate projects. By choosing projects with higher chances of InnerSource success, you also increase the chances of the overall InnerSource program to be successful.
```

I would trie to tie this back more immediately to the problem, and describe "What is the situation after the problem has been solved?" If we assume that the Problem exists in a situation where an InnerSource Program is launched, let's talk about that.

I made a suggestion above. However let's also check if the other sentences talk about the "situation after the problem has been solved" as well.